src/index.d.ts

```diff
@@ -0,0 +1,73 @@
+// TODO(kjin): Unify these definitions with those of the Debugger Agent.
```

Comment:
I would prefer to call this file something different since it is not actually related to the module entrypoint.

Reply:
Done